Doc/library/ast.rst

```diff
@@ -1563,7 +1563,7 @@ and classes for traversing abstract syntax trees:
    Safely evaluate an expression node or a string containing a Python literal or
    container display.  The string or node provided may only consist of the
    following Python literal structures: strings, bytes, numbers, tuples, lists,
-   dicts, sets, booleans, and ``None``.
+   dicts, sets, booleans, ``None`` and ``Ellipsis``.
```

Comment:
Let's also take this opportunity to be more specific about what is meant by "numbers".  Replace that  with "ints, floats, complex numbers".

Also, please make the same update to the docstring for *ast.literal_eval()*.